Modules/_testbuffer.c

```diff
@@ -2813,48 +2813,27 @@ static struct PyMethodDef _testbuffer_functions[] = {
     {NULL, NULL}
 };
 
-static struct PyModuleDef _testbuffermodule = {
-    PyModuleDef_HEAD_INIT,
-    "_testbuffer",
-    NULL,
-    -1,
-    _testbuffer_functions,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
-
-
-PyMODINIT_FUNC
-PyInit__testbuffer(void)
+static int
+_testbuffer_exec(PyObject *m)
 {
-    PyObject *m;
-
-    m = PyModule_Create(&_testbuffermodule);
-    if (m == NULL)
-        return NULL;
-
-    Py_SET_TYPE(&NDArray_Type, &PyType_Type);
-    Py_INCREF(&NDArray_Type);
-    PyModule_AddObject(m, "ndarray", (PyObject *)&NDArray_Type);
+    if (PyModule_AddType(m, &NDArray_Type) < 0)
+        return -1;
```

Comment:
PEP 7 coding style requires braces for new code:

```suggestion
    if (PyModule_AddType(m, &NDArray_Type) < 0) {
        return -1;
    }
```

Same remark for code below. Please also add braces in the if that you modify.